common/app/services/ConfigAgentTrait.scala

```diff
@@ -108,18 +107,14 @@ trait ConfigAgentTrait extends ExecutionContexts with Logging {
 
   def getFrontPriorityFromConfig(pageId: String): Option[FrontPriority] = {
     configAgent.get() flatMap {
-      _.fronts.get(pageId) map {
-        _.priority match {
-          case Some("commercial") => CommercialPriority
-          case Some("training") => TrainingPriority
-          case _ => EditorialPriority
-        }
+      _.fronts.get(pageId) map { frontJson =>
+        Front.fromFrontJson(pageId, frontJson).priority
```

Comment:
Ah yes. Great! There might be similar stuff like this hanging around because of the history of where all this code resided